Makefile

```diff
@@ -70,6 +70,10 @@ vendor:
 	$(GO) mod vendor
 	$(GO) mod verify
 
+.PHONY: fmt
+fmt:
+	$(GO) fmt $(go list ./... | grep -v /vendor/)
```

Comment:
we've historically used `goimports` in OLM and operator-registry.

Reply:
That's fine with me - I mainly used `gofmt` due to familiarity. The only potential problem is that I've seen `goimports` take upwards of 20 minutes for larger codebases, but it's possible I needed to further refine what paths I was passing to it.

Reply:
Updated - had to wrangle a bit with how `go get ...` manipulates the go.mod and go.sum and disable module awareness for that single command. I can add a comment for why that's needed if that behavior is confusing and needs more clarification.